### ISSUE TITLE
Send to all contacts

### ISSUE THREAD

**Issue by rocosttaa** [Briuor/wbm#33]
How to send to all my contacts at once?
It is possible?

**Comment by Guyc1800** [comment]
> How to send to all my contacts at once? It is possible?
no, just make a list of all the numbers and put it in the wbm.send function
